src/libDirectoryService/PoWProcessing.cpp

```diff
@@ -42,15 +42,136 @@
 using namespace std;
 using namespace boost::multiprecision;
 
-bool DirectoryService::ProcessPoWSubmission(
+bool DirectoryService::SendPoWPacketSubmissionToOtherDSComm() {
```

Comment:
Since we do both sending of packet and processing this packet ourselves as well, maybe rename the function?